src/as_execution/abi.rs

```diff
@@ -930,6 +930,23 @@ where
     Ok(buffer)
 }
 
+/// performs a sha256 hash on bytes and returns the hash as bytes
```

Comment:
Maybe specify what is return as a hash: hex digest or something else? with an example

Reply:
Yes, I'll add a concret example in the doc

Reply:
I tried to make a clear description to this method. May be we'll add integration tests with an SC.